pool_configs/vot.json

```diff
@@ -0,0 +1,87 @@
+{
+    "enabled": false,
+    "coin": "vot.json",
+
+    "address": "",
+    "_comment_address": "a transparent address to send coinbase rewards to and to transfer to zAddress.",
+
+    "zAddress": "",
+    "_comment_zAddress": "a private address used to send coins to tAddress.",
+
+    "tAddress": "",
+    "_comment_tAddress": "transparent address used to send payments, make this a different address, otherwise payments will not send",
+
+    "walletInterval": 2.5,
+
+    "rewardRecipients": {
+        "": 0.2
+    },
+
+    "paymentProcessing": {
+	"minConf": 10,
+	"enabled": false,
+	"paymentMode": "prop",
+	"_comment_paymentMode":"prop, pplnt",
+	"paymentInterval": 20,
+	"minimumPayment": 0.1,
+	"maxBlocksPerPayment": 3,
+	"daemon": {
+        "host": "127.0.0.1",
+        "port": 8232,
```

Comment:
Update rpc ports to their defaults. xsg i believe is 16112. they can be found on the coins github repo or usually their announcement page on bitcointalk. This will need done for the other pool_configs.